### ISSUE TITLE
Extended phone support

### ISSUE THREAD

**Issue by nravic** [everestpipkin/image-scrubber#31]
It might also be worth the effort to quickly port over some of the code to react-native or any other js cross-platform framework, especially considering most people using this would be doing so on their phones. 

Let me know what you think and I could whip something up quickly!

**Comment by everestpipkin** [comment]
I'm not really a mobile developer - can you clarify for me what a react-native port would add for us? Would it be so it can actually run as an application, not just a website?

**Comment by nravic** [comment]
Yup, exactly. Putting it onto react native (or any other js web app framework) enables us to essentially compile js code down to native code (for both Android and iOS (this is very hand-wavey, there's a little more to it)). 

It'll run as an application on the phone, so there's a little less hesitation re: uploading images to scrub. Also I think less of a barrier to entry

**Comment by everestpipkin** [comment]
I agree on the barrier to entry part, its sad but true that websites feel clunky on phones. Have you been through an app store approval process? What do you think the chances are of us getting approved? I can't really imagine why they'd say no as the functionality is really quite simple, except that it is so expressly a protest tool.

**Comment by bMacSwigg** [comment]
As a general comment -- which you may already be thinking about, so sorry if so -- I think there's a balance to be struck between immediate functionality & long-term functionality.
* In the short term, is it feasible to get an app approved while the current George Floyd protests are still happening? (Maybe, maybe not -- it's not a process I'm familiar with, and it's anyone's guess how long the protests will continue)
* In the long term, having an option to use the image scrubber as an app would be more usable and accessible, and regardless of the answer to the above seems like a worthwhile goal

**Comment by nravic** [comment]
It's worth an attempt for approval. Otherwise, it's a non-issue on Android devices. We can just ship the apk (by providing a link to it) and they can download & install like you would an exe on Windows. 

I've been through the android approval process and the only issue is time, since it usually takes a week or so for them to get back to you. Currently going through the iOS process, so will let you know about that.

**Comment by caelinsutch** [comment]
Flutter is a great option for mobile, bit faster and easier to write than React Native, but I'll help either way.  Android and iOS process is usdually pretty quick for me, I've shipped a few apps.
